packages/ketcher-core/src/application/render/renderers/PolymerBondRenderer.ts

```diff
@@ -728,7 +728,8 @@ export class PolymerBondRenderer extends BaseRenderer {
           .attr('stroke', '#57FF8F')
           .attr('stroke-width', 2)
           .attr('fill-opacity', 0)
-          .attr('d', this.path);
+          .attr('d', this.path)
+          .attr('class', 'snake-bond-selection');
```

Comment:
You can just replace this line with `.attr('class', 'dynamic-element')` and I think it will resolve the issue and make the logic easier